src/cocoa/toga_cocoa/libs/appkit.py

```diff
@@ -303,15 +303,18 @@ def NSColorUsingColorName(background_color):
 # NSImage.h
 NSImage = ObjCClass('NSImage')
 
-NSImageAlignCenter = 0
-NSImageAlignTop = 2
-NSImageAlignTopLeft = 3
-NSImageAlignTopRight = 4
-NSImageAlignLeft = 5
-NSImageAlignBottom = 6
-NSImageAlignBottomLeft = 7
-NSImageAlignBottomRight = 8
-NSImageAlignRight = 9
+
+class NSImageAlignment(Enum):
+    Center = 0
+    Top = 1
+    TopLeft = 2
+    TopRight = 3
+    Left = 4
+    Bottom = 5
+    BottomLeft = 6
+    BottomRight = 7
+    Right = 8
```

Comment:
These numbers look like they've got out of alignment with the original constants...

Reply:
How so?

Reply:
/me looks sheepish... for some reason, I thought this patch was *adding* some constants; but the whole point is to fix the values. Problem exists between my keyboard and chair :-)

Reply:
No worries, that happens to all of us :)